modules/etcd-cert-alerts-metrics-signer.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * security/certificate_types_descriptions/etcd-certificates.adoc
+
+:_mod-docs-content-type: CONCEPT    
+[id="etcd-cert-alerts-metrics-signer_{context}"]
+== etcd certificate rotation alerts and metrics signer certificates
+
+Two alert types inform users about pending `etcd` certificate expiration:
+[horizontal]
+`etcdSignerCAExpirationWarning`:: Occurs 730 days until the signer expires.
+`etcdSignerCAExpirationCritical`:: Occurs 365 days until the signer expires.
+
+You can rotate the certificate:
+
+* When you receive an expiration alert
+* When the private key is leaked
+
+NOTE: When a private key is leaked, you must rotate all of the certificates.
```

Comment:
Honestly wonder if this is more important than a note :thinking: 

```suggestion
[NOTE]
====
When a private key is leaked, you must rotate all of the certificates.
====
```

Reply:
@jeana-redhat I can't find it now where I saw this construct for an admonition documented (but I will!). However, I will make this change and also change it to 'IMPORTANT', because I agree that it is more impacting than having it as a note.